packages/nds-card/nds-card.d.ts

```diff
@@ -13,6 +13,7 @@ declare module "@nice-digital/nds-card" {
 
 	export interface CardProps {
 		headingText: string;
+		containerElementType: React.ElementType;
 		elementType?: React.ElementType;
```

Comment:
`elementType` now makes sense to be called `headingElementType` or something

Reply:
have changed `elementType` for the container and `headingElementType` for the heading